dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -127,6 +128,7 @@ internal class RumResourceScope(
 
         val finalTiming = timing
         val duration = eventTime.nanoTime - startedNanos
+        val isFirstParty = CoreFeature.firstPartyHostDetector.isFirstPartyUrl(url)
```

Comment:
Same here...can't we just inject the `firstPartyHostDetector` here and keep it as a singleton in the `CoreFeature` ?